e2e/e2e_test.go

```diff
@@ -0,0 +1,169 @@
+package e2e
+
+import (
+	"fmt"
+	"log"
+	"testing"
+	"time"
+
+	whbar "github.com/limechain/hedera-eth-bridge-validator/app/clients/ethereum/contracts/whbar"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	common "github.com/ethereum/go-ethereum/common"
+	"github.com/hashgraph/hedera-sdk-go"
+	ethclient "github.com/limechain/hedera-eth-bridge-validator/app/clients/ethereum"
+	tx "github.com/limechain/hedera-eth-bridge-validator/app/process/model/transaction"
+	"github.com/limechain/hedera-eth-bridge-validator/config"
+	validatorproto "github.com/limechain/hedera-eth-bridge-validator/proto"
+	"google.golang.org/protobuf/proto"
+)
+
+func Test_E2E(t *testing.T) {
```

Comment:
nit:
I think that we could categorise the e2e test into 3 categories:
- Verify Crypto Transfer
- Verify Topic signatures
- Verify Eth TX execution

That being said we could create 3 functions for that -> 
- line 46 - 84 to be `verifyCryptoTransfer`
- line line 85 - 129 to be `verifyTopicMessages`
and the rest -> `verifyEthereumTXExecution`